### ISSUE TITLE
[Bug] Cannot create listener

### ISSUE THREAD

**Issue by lolcatlolcat** [cobbr/Covenant#162]
**Bug**
Fresh install of Covenant on Ubuntu 18.04 results in a stack trace/error message when trying to create a listener within the UI. As a result, cannot create a listener of any kind.

**To Reproduce**
1. Follow install instructions [here](https://github.com/cobbr/Covenant/wiki/Installation-And-Startup#option-1---dotnet-core)
2. Click on 'Listeners' in the left-hand sidebar
3. Click on 'Create' on the Listeners page

**Expected behavior**
To be able to create a listener in the UI.

**Screenshots**
![image](https://user-images.githubusercontent.com/25574768/79489781-d07a3980-7fe9-11ea-99e6-bfade038f4fb.png)
![image](https://user-images.githubusercontent.com/25574768/79489847-ef78cb80-7fe9-11ea-99d0-97c9ddbd1ece.png)
![image](https://user-images.githubusercontent.com/25574768/79489912-0b7c6d00-7fea-11ea-86d0-36b7bb9a334e.png)

**Covenant Server Information:**
 - OS: Ubuntu 18.04
 - Native

**Browser Information:**
 - Chrome and Edge
 - Version 81.0.4044.113 (Official Build) (64-bit) | Version 81.0.416.53 (Official build) (64-bit)

**Comment by cobbr** [comment]
Additional context:
```
An unhandled exception occurred while processing the request.
NullReferenceException: Object reference not set to an instance of an object.
Covenant.Views.Listener.Views_Listener__CreateBridgeListener+<>c__DisplayClass20_0+<<ExecuteAsync>b__2>d.MoveNext() in _CreateBridgeListener.cshtml, line 44
```

**Comment by cobbr** [comment]
@lolcatlolcat I think this should be fixed in the new version 79c45e7

Can you verify this problem is fixed for you?

**Comment by cobbr** [comment]
Closing for now, feel free to reopen if you continue to have problems.
